frontend/coffee/profile.coffee

```diff
@@ -1,3 +1,7 @@
+get = (name) -> document.getElementById(name).value
```

Comment:
If you want syntactic sugar for this, I think it's equivalent to `$('#name').value`, which is probably more recognizable for Javascript/Coffescript programmers. See the `$('#password-form')` example.

Reply:
I think I had problems getting jQuery to work there initially, so I went with plain CoffeScript/JavaScript. I'll try again if it works now.

Reply:
I think I got it to work.